django-stubs/test/runner.pyi

```diff
@@ -159,12 +182,23 @@ class DiscoverRunner:
     def get_databases(self, suite: TestSuite) -> set[str]: ...
     def run_tests(self, test_labels: list[str], extra_tests: list[Any] | None = ..., **kwargs: Any) -> int: ...
 
-def is_discoverable(label: str) -> bool: ...
+def try_importing(label: str) -> bool: ...
+def find_top_level(top_level: AnyStr) -> AnyStr: ...
+def shuffle_tests(tests: list[TestCase], shuffler: Shuffler) -> Iterator[TestCase]: ...
+def reorder_test_bin(
+    tests: list[TestCase], shuffler: Shuffler | None = ..., reverse: bool = ...
+) -> Iterator[TestCase]: ...
+def reorder_tests(
+    tests: list[TestCase], classes: tuple[type[TestCase], ...], reverse: bool = ..., shuffler: Shuffler | None = ...
+) -> Iterator[TestCase]: ...
```

Comment:
Why do you use `Iterator` here and `Generator` in `iter_test_cases`? They should be similar